### ISSUE TITLE
trying to modify the ooxml of a word using api

### ISSUE THREAD

**Issue by XWANG0224** [OfficeDev/office-js#5062]
I'm trying to modify the pgMar part of the xml of a word document, 
the code is like this 
`const getXMLOfSections = async () => {
        try{
            await Word.run(async (context) => {
                const bodyxml = context.document.body.getOoxml();
                await context.sync();
                const xml = bodyxml.value;
                const parser = new DOMParser();
                const xmlDoc = parser.parseFromString(xml, "text/xml");
                const pgMar = xmlDoc.getElementsByTagName("w:pgMar")[0];
                // console.log(pgMar);
                pgMar.setAttribute("w:top", "100");
                pgMar.setAttribute("w:right", "100");
                pgMar.setAttribute("w:bottom", "100");
                pgMar.setAttribute("w:left", "100");
                console.log(pgMar);
                const serializer = new XMLSerializer();
                const newXML = serializer.serializeToString(xmlDoc);
                context.document.body.insertOoxml(newXML, Word.InsertLocation.replace);
                await context.sync();
            })
        } catch (error) {
            console.error(error);
        }
    }`
after the setattribute function, the xml is successfully updated, but when I try to apply the new xml to the doc using insertOoxml, nothing changes, I wonder why is this happening?

**Comment by RuizhiSunMS** [comment]
Hi @XWANG0224 , thx for your submit.
Would you please confirm your parseFromString did return a xml object? In theory, microsoft word has a special mime type which is not supported by domparser.
Btw, we will put it on our backlog #9518612, the team will investigate and we will reply to you as soon as there is any progress. Thank you for your patience.

**Comment by XWANG0224** [comment]
yes, I'm sure the right xml object is returned because the following setAttribute functions works just as expected

**Comment by RuizhiSunMS** [comment]
> yes, I'm sure the right xml object is returned because the following setAttribute functions works just as expected

ACK. The ADO has been repathed into Word team's backlog.
